_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+build affinity towards the new name from scratch. We weren't sure what to expect
+when we announced the rename. What followed, was an outpouring of support and
+shock from the larger developer community. 
+
+<p align="center">
+ <a href="https://twitter.com/trinodb/status/1343330429684703232?s=20" target="_blank">
+   <img align="center" width="100%" src="/assets/blog/2021-review/tweets.png"/>
+ </a>
+</p>
+
+<!--more-->
+
+The community immediately got to work. The project had to change the namespace
+practically overnight from the `io.prestosql` namespace to `io.trino` and
+a [migration blog post]({{site.url}}/blog/2021/01/04/migrating-from-prestosql-to-trino.html)
+was published. We reached out to multiple projects to update compatibility to
+Trino.
+
+* [DBeaver](https://github.com/dbeaver/dbeaver/pull/10925)
+* [QueryBook](https://github.com/pinterest/querybook/issues/509)
+* [Homebrew](https://github.com/Homebrew/homebrew-core/pull/83185)
+* [dbt](https://github.com/dbt-labs/dbt-presto/issues/39)
+* [sqlalchemy](https://github.com/dungdm93/sqlalchemy-trino/issues/20)
+* [sqlpad](https://github.com/sqlpad/sqlpad/pull/974)
+* [Apache Superset](https://github.com/apache/superset/pull/13105)
+* [Redash](https://github.com/getredash/redash/pull/5411)
+* [Awesome Java](https://github.com/akullpp/awesome-java/pull/917)
+* [Awesome For Beginners](https://github.com/MunGell/awesome-for-beginners/pull/933)
+* [Airflow](https://github.com/apache/airflow/pull/15187)
+* [trino-gateway](https://github.com/lyft/presto-gateway/issues/134)
+* [Metabase](https://github.com/metabase/metabase/issues/17532)
+* and so much more...
+
+Not only was the community excited and supportive about the brand change, but
+particularly they were all loving the new mascot. Our adorable bunny was soon 
+after [named Commander Bun Bun by the community](/episodes/10.html).
+
+<p align="center">
+ <a href="https://twitter.com/jtannady/status/1346888143459545092" target="_blank">
+   <img align="center" width="50%" src="/assets/blog/2021-review/cbb.png"/>
+ </a>
+</p>
+
+Immediately after the announcement, the [Trino GitHub repository](https://github.com/trinodb/trino)
+started trending in number of stargazers. Up until this point, the PrestoSQL
+GitHub repository had only amassed 1,600 stargazers in the two years since it 
+had split from the PrestoDB repository. However, within four months after the 
+renaming, the number of stargazers had doubled. GitHub stars, issues, pull 
+requests and commits started growing at a new trajectory.
+
+<p align="center">
+ <a href="https://twitter.com/bitsondatadev/status/1344028682126565381" target="_blank">
+   <img align="center" width="50%" src="/assets/blog/2021-review/trending.png"/>
+ </a>
+</p>
+
+## Community growth
+
+While we are on the subject, lets talk about the health of the Trino community.
+As mentioned the GitHub stargazers trajectory skyrocketed. At the time of 
```

Comment:
"As mentioned above". 

However, I think this paragraph is too repetitive. The numbers were just discussed in the section above. I would either some of the details that weren't discussed to the previous section and just get rid of this paragraph. Maybe just keep the intro sentence.